### DISCUSSION TITLE
A bug when switching between GlobeView and MapView?

### DISCUSSION THREAD

**Discussion by yanshjie** [visgl/deck.gl#7472] [Q&A] [answered]
When switching between GlobeView and MapView, the MVTlayer didn't show up in the same zoom level.  To be more precise, when my MVTlayer renders in MapView in certain zoom level for example 7, and when I changed view to GlobeView by using `deck.setProps({view: ...})`, the MVTlayer did not show up in GlobeView in zoom level 7, but other zoom level is ok.

The docs says there are some known rendering issues when  switching between the two views. So I am eager to know whether this indeed a bug of deckgl, if so, is there a way to workaround it?

![图片](https://user-images.githubusercontent.com/98957105/205539666-1c14cf8f-6423-4e88-8c71-b4fd0587a2bf.png)

**Comment by Pessimistress** [comment, marked as answer]
https://deck.gl/docs/developer-guide/views#rendering-layers-in-multiple-views

TileLayer/MVTLayer does not support multiple views. If you switch to a different view you need to create a new layer (i.e. change its id).
